src/mame/snk/hng64_v.cpp

```diff
@@ -1468,7 +1475,15 @@ void hng64_state::tcram_w(offs_t offset, uint32_t data, uint32_t mem_mask)
 		m_screen_dis = 0;
 
 		visarea.set(min_x, min_x + max_x - 1, min_y, min_y + max_y - 1);
-		m_screen->configure(HTOTAL, VTOTAL, visarea, m_screen->frame_period().attoseconds());
+
+		// TODO: properly calculate this from screen params
+		attoseconds_t period;
+		if (max_y == 448)
+			period = HZ_TO_ATTOSECONDS(59.430077); // everything apart from fatfurwa uses a 512x448 resolution, and 59.43hz appears to sync with hardware
+		else
+			period = HZ_TO_ATTOSECONDS(61.651673); // fatfurwa uses 512x432, sync frequency not verified
```

Comment:
Is 59.43-ish Hz regular NeoGeo refresh rate, pumped to interlace?
fatfurwa refresh rate is wild, I guess they just shortened 16 display lines in order to get that refresh rate.
Bottom line: this may just have a (24 MHz / 4) clock, which we need to multiply by 2 for MAME limit on interlace. Is there any candidate on board?

Reply:
the info we have doesn't list a 24Mhz.

It wouldn't surprise me if the true video params were somewhere near the raster interrupt config registers, not here in the 'tc' part anyway, tc should just be the mixer (and might really only deal with palette+ mixing)